### DISCUSSION TITLE
Question two conection shaft, not a the same speed

### DISCUSSION THREAD

**Discussion by VictorRenkin** [petrobras/ross#1117] [Q&A]
I’m using ROSS for one of my courses, and we’re working with a twin-spool turbojet engine. 
![Capture](https://github.com/user-attachments/assets/82f1fd3a-5886-4622-9bdb-0bd082fedde0)
The engine has two shafts that run at different speeds with a linearize relations, and I’d like to connect them to create the complete engine. However, I’m not sure how to proceed. I have a roller bearing for the connection between the two shafts, but I’m uncertain about how to plot it or use it with Campbell diagrams.  If anyone has suggestions on how to approach this, I’d appreciate it—I’m still a beginner with ROSS.

**Comment by jguarato** [comment]
Hi @Renkinios!

We are currently developing a new element class called `MultiRotor`. While the initial goal of this class is to couple two shafts using gears—slightly different from the problem you are addressing—I believe the concept could be adapted to your case. Although the `MultiRotor` class is not yet available in the latest release of ROSS, you can follow its progress in PR #1111. In the future, we may extend the functionality of this class to support the type of shaft connection you're working with.
